dynamiq/nodes/tools/firecrawl.py

```diff
@@ -147,6 +147,6 @@ def execute(
                 "warning": scrape_result.get("data", {}).get("warning"),
             }
 
-        logger.debug(f"Tool {self.name} - {self.id}: finished with result {str(result[:200])}...")
+        logger.debug(f"Tool {self.name} - {self.id}: finished with result {str(result)[:200]}...")
```

Comment:
it will break if there's less than 200 elements

Reply:
Nope:
```
>>> a = "12345"
>>> a[:10]
'12345'
>>> a[:1000]
'12345'
```